### PR TITLE
#92 SortCommand feature

### PR DESCRIPTION
**Pull request by neilish3re**
#92 

Allows the user to sort coin list in either ascending or descending alphabetical order.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.commands;
+
+/**
+ * Sorts all coins in Coin List by alphabetical order and displays sorted list to user
+ */
+//@@author Neil
```
Comment (ewaldhew) [inline src/main/java/seedu/address/logic/commands/SortCommand.java]: Change these to `@@author neilish3re`

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+public class SortCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "sort";
```
Comment (ewaldhew) [inline src/main/java/seedu/address/logic/commands/SortCommand.java]: COMMAND_WORD = "order"
COMMAND_ALIAS = "o"

**Comment by Eldon-Chung** [comment]
A new line at the end of files SortCommand and SortCommandParser is needed.

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * Sorts all coins in Coin List by alphabetical order and displays sorted list to user
+ */
+//@@author neilish3re
+public class SortCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "order";
+    public static final String COMMAND_ALIAS = "o";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all the coins the user has input into the Coin Book as a list"
```
Comment (ewaldhew) [inline src/main/java/seedu/address/logic/commands/SortCommand.java]: Usage message should include description of parameters

Also update User Guide

**Pull request merged**